src/EditorFeatures/Core/Implementation/IntelliSense/ModelComputation.cs

```diff
@@ -96,7 +96,7 @@ public TModel WaitForController()
             var model = ModelTask.WaitAndGetResult(CancellationToken.None);
             if (!_notifyControllerTask.IsCompleted)
             {
-                OnModelUpdated(model, updateController: true);
```

Comment:
what's the intuition around removing this value.

Reply:
⏱️ Trying to restore this value